api/nodes/views.py

```diff
@@ -1367,6 +1371,168 @@ def get_object(self):
         return fobj
 
 
+class NodeAddonList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin, NodeMixin):
+    """List of addons connected to this node *Read-only*
+
+    TODO: More Docs
+    """
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        ContributorOrPublic,
+        ExcludeRetractions,
+        base_permissions.TokenHasScope,
+    )
+
+    serializer_class = NodeAddonSettingsSerializer
+    view_category = 'nodes'
+    view_name = 'node-addons'
+
+    def get_default_queryset(self):
+        qs = []
+        for addon in ADDONS_OAUTH:
+            addon_settings = ADDONS_AVAILABLE_DICT[addon].settings_models.get('node', None)
+            obj = None
+            enabled = False
+            try:
+                obj = addon_settings.find(Q('owner', 'eq', self.get_node()._id) & Q('deleted', 'eq', False))[0]
+                enabled = True
+            except IndexError:
```

Comment:
It could be argued the the `enabled = True` belongs in an `else` block in this try/except. I don't have strong feelings about this but it's worth thinking about.